AutoGear.lua

```diff
@@ -2041,7 +2054,7 @@ AutoGearFrame:SetScript("OnEvent", function (this, event, arg1, arg2, arg3, arg4
 				end
 			end
 			--available quests
-			if not (IsClassic or IsTBC or IsWotLK) then
+			if TOC_VERSION_CURRENT >= 90001 then
```

Comment:
What's the rationale for 90001 instead of 90000?  I ask because I'd like to use variables for expansion numbers.

Reply:
version of game, when this method appeared

Reply:
i think i was too precise, and if this addon use only of official clients, then accurate numbers will not be neccessary